YoutubeExplode/Videos/Streams/IStreamInfo.cs

```diff
@@ -0,0 +1,50 @@
+using System.Collections.Generic;
+using System.Linq;
+
+namespace YoutubeExplode.Videos.Streams
+{
+    /// <summary>
+    /// Generic YouTube media stream.
+    /// </summary>
+    public interface IStreamInfo
+    {
+        /// <summary>
+        /// Stream tag.
+        /// Uniquely identifies a stream inside a manifest.
+        /// </summary>
+        int Tag { get; }
+
+        /// <summary>
+        /// Stream URL.
+        /// </summary>
+        string Url { get; }
+
+        /// <summary>
+        /// Stream container.
+        /// </summary>
+        Container Container { get; }
+
+        /// <summary>
+        /// Stream size.
+        /// </summary>
+        FileSize Size { get; }
+
+        /// <summary>
+        /// Stream bitrate.
+        /// </summary>
+        Bitrate Bitrate { get; }
+    }
+
+    /// <summary>
+    /// Extensions for <see cref="IStreamInfo"/>.
+    /// </summary>
+    public static class StreamInfoExtensions
+    {
+        /// <summary>
+        /// Gets the stream with highest bitrate.
+        /// Returns null if sequence is empty.
+        /// </summary>
+        public static IStreamInfo WithHighestBitrate<T>(this IEnumerable<IStreamInfo> streamInfos) =>
```

Comment:
I think the <T> might be unnecessary here.

Reply:
you're right, thanks